src/BuildOnWindows.cmake

```diff
@@ -21,7 +16,6 @@ add_library(
   ${ATen_XPU_NATIVE_CPP_SRCS}
   ${ATen_XPU_GEN_SRCS})
 install(TARGETS torch_xpu_ops_aten DESTINATION "${TORCH_INSTALL_LIB_DIR}")
-# target_compile_definitions(torch_xpu_ops_aten PRIVATE CAFFE2_BUILD_MAIN_LIB)
```

Comment:
Could you elaborate why we needed it before and why we deprecate it now?

Reply:
It was already commented out, so I removed it. It wasn't used before too.